uri-schemes.md

```diff
@@ -124,6 +126,90 @@ HTTP Status code: NOT 2XX, idealy the 4XX if it's user error and 5XX if it's the
 
 json object can have other properties too, but currently they are ignored by core.
 
+## **DCLOGIN** <a name="DCLOGIN"></a>
+
+|                          |                       |
+| ------------------------ | --------------------- |
+| Scheme                   | `DCLOGIN:`            |
+| Used for                 | Account setup         |
+| Related Terms\*          | Account Login QR code |
+| Available on             | draft                 |
+| Decoded by the core \*\* | draft                 |
+| Other apps using it      | none, only DeltaChat  |
+
+### Syntax
+
+```
+dclogin:user@host?p=password&v=1[&options]
+dclogin:user@host/?p=password&v=1[&options]
+dclogin://user@host/?p=password&v=1[&options]
+# example: (email: me@example.com, password: securePassword)
+dclogin://me@example.com?p=securePassword&v=1
+# example: (email: myself@example.com, password: url/Encoded\@passw@rd)
+dclogin://myself@example.com?p=url%2FEncoded%5C%40passw%40rd&v=1
+# example: (email: myself@example.com, password: 123456, insecure smtp at different server)
+dclogin://myself@example.com?p=123456&v=1&sh=mail.example.com&sc=3&ss=plain
```

Comment:
I think you're diverging a bit far from [the URI syntax](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier#syntax).

I'd refer to the URI spec and say:
* must have `userinfo`
* must have `host`
* must be single `/` for path
* query parameters ad defined

This means example 1 and 2 should be considered invalid.

Reply:
so `path` is allowed to be emtpy, may as well omit `/` in that case.

Reply:
So force using `://` and no / for the path?
It is parsed by an URL parser lib we already use in core so path is just ignored anyway but we could still remove that example from the spec.

Reply:
That the existing parser lib parses it is already great.  As usual being strict in the spec but liberal in parsing is a good thing probably.

Yes to requiring `://` as separator (though if parsing without works then that's great), so I'd remove the examples without it.  All the other examples are already following the URI spec I think.

And maybe it's also worth specifically describing the syntax wrt to URI schema.  Something like:
```
The generic URI schema is:

URI = scheme ":" ["//" authority] path ["?" query] ["#" fragment]
authority = [userinfo "@"] host [":" port]

For DCLOGIN this means:
- *scheme* is always dclogin
- There must be an *authority* section
  - The *userinfo* part is required.
- The *path* may be omitted or be a single `/`.
- The fragment must be omitted.
- The query parameters ... (as currently described)
```
The benefit of this is that it explicitly acknowledges the URI format and signals that we're aiming to be compatible.  Though I won't mind too much if you don't agree with this.